apps/sms/desktop-mock/navigator_moz_mobilemessage.js

```diff
@@ -1116,7 +1116,7 @@
           id: messagesDb.id++,
           type: 'sms',
           read: false,
-          timestamp: now,
+          timestamp: now + 1,
```

Comment:
Ha-ha that bothered me a lot, but never allocated time to find out what was wrong :)